monitoring/uss_qualifier/resources/interuss/mock_uss.py

```diff
@@ -29,12 +29,20 @@ def get_status(self) -> fetch.Query:
         return fetch.describe_query(resp, initiated_at)
 
     # TODO: Add other methods to interact with the mock USS in other ways (like starting/stopping message signing data collection)
+    def start_msg_sign_recording(self) -> fetch.Query:
+        initiated_at = arrow.utcnow().datetime
+        resp = self.session.post("/scdsc/v1/startreport", scope=SCOPE_SCD_QUALIFIER_INJECT)
```

Comment:
mock_uss doesn't implement this endpoint yet, nor is this endpoint defined -- I think we need to define and implement it first before we enable a client to call it.

Reply:
That's right. That's why we had that big PR, which had all the changes together. As you wanted separate PR, we couldn't put in this PR.

Reply:
I don't think the answer is a big PR -- instead, I think we'd want to merge a small PR for the dependency first before starting the PR for something that requires that dependency (since a PR with something that requires a dependency that doesn't exist would necessarily introduce an error/problem, and we don't want to merge anything to master that we don't think is fully working, though NotImplementedErrors without test or behavior regressions are often ok as building blocks when practical).

The "Prototyping" section of [the second article linked from our CONTRIBUTING.md with regard to large PRs](https://essenceofcode.com/2019/10/29/the-art-of-small-pull-requests/) seems relevant here.

Reply:
Sure, I think then we ll need to review the message-signing PR before this one - https://github.com/interuss/dss/pull/892. This one has mock uss endpoints implemented. Would that be ok?